tests/tests_fabric/strategies/test_xla_fsdp.py

```diff
@@ -0,0 +1,232 @@
+# Copyright The Lightning AI team.
```

Comment:
I recommend that we split this file into unit tests and integration tests. See our files in `test_fsdp.py` and `test_fsdp_integration.py` (i.e., the save-load test would be a integration test).

Reply:
I will split these

Reply:
Done